docs/notes.md

```diff
@@ -0,0 +1,353 @@
+
+- all motr and s3 addb are collected on individual process basis on each node
+- process PID is a part of name
+- during the conversion from bin to text, process PID HAVE TO BE SAVED as a part of a filename. Processing scripts assume special filename format
+    - filename format: `<name>_PID.txt` where `<name>` should not contain `-` or `_`
+    - suggested name for motr is `dumps_<PID>.txt`
+    - suggested name for s3 is `dumpc_<PID>.txt`
+
+- s3server addb plugin - `/opt/seagate/cortx/s3/addb-plugin/libs3addbplugin.so` - should only be used with the version of s3serve it was build for
+- `m0addb2dump` is build and deployed with mero
+
+Steps:
+
+- stop ALL s3server services
+- stop ALL motr services
+- collect motr addb on dev env
+
+```
+cd ./third_party/motr/addb2/
+# motr addb
+dump_s="/var/motr/systest-*/ios*/addb-stobs-*/o/100000000000000:2"
+for d in $dump_s; do
+    pid=$(echo $d | sed -E 's/.*addb-stobs-([a-z0-9]*)[/].*/\1/')
+    echo 'mero '${pid}
+    ./m0addb2dump -f -- "$d" > "$collect_addb/dumps_${pid}.txt"
+done
+cd -
+```
+
+- collect s3server addb on dev env
+
+```
+cd ./third_party/motr/addb2/
+# s3server addb
+dump_s="/var/log/seagate/motr/s3server-*/addb_*/o/100000000000000:2"
+for d in $dump_s; do
+    pid=$(echo $d | sed -E 's/.*addb_([0-9]+)[/].*/\1/')
+    echo 's3server '${pid}
+    ./m0addb2dump -f -p /opt/seagate/cortx/s3/addb-plugin/libs3addbplugin.so -- "$d" > "$collect_addb/dumpc_${pid}.txt"
+done
+cd -
+```
+
+- where `$collect_addb` is output folder
+- in case of deployed cluster addb should be collected for each node
+- during the process lots of m0trace files could be created. it is better to delete them - they require lots of disk space
+
+## Processing
+
+Addb tools repo: `https://github.com/Seagate/seagate-tools`
+Addb scripts: `https://github.com/Seagate/seagate-tools/tree/main/performance/PerfLine/roles/perfline_setup/files/chronometry_v2`
+Mainteiner: Alexander Sukhachev
+Knows a lot: Maxim Malezhin
+Also: Ivan Tishchenko, Anatoliy Bilenko
+
+- convert text to db
+
+```
+#!/usr/bin/env bash
+set -ex
+script_dir=seagate-tools/performance/PerfLine/roles/perfline_setup/files/chronometry_v2
+raw_data_dir=addb_res    # path to folder with test results
+raw_data_dir=$(realpath -e $raw_data_dir)
+cd $script_dir
+for d in $(ls -d $raw_data_dir/addb_c*); do
+    test_name=$(echo $d | sed -E 's/.*addb_(c[0-9]+_s[0-9]+[mMkKgGbB]+_cnt[0-9]+_d[0-9]+)/\1/')
+    echo "$d :> $test_name"
+    ./addb2db_multiprocess.sh --dumps $d/dump*.txt
+    cp ./m0play.db $d/m0play_${test_name}.db
+    rm -rf ./m0play.db
+done
+cd -
+```
+
+- example of `raw_data_dir` folder structure
+    - folder `raw_data_dir`
+        - subfolder `addb_c1_s16Mb_cnt1000`
+            - content `dumps_<PID>.txt`
+            - content `dumpc_<PID>.txt`
+        - subfolder `addb_c22_s512Kb_cnt1000`
+            - content `dumps_<PID>.txt`
+            - content `dumpc_<PID>.txt`
+
+- it is possible to add S3Authserver logs - `app.log` - to DB
```

Comment:
Can you provide full command to add S3AuthServer log to DB?

Reply:
it looks like in the latest version it is not supported, so deleting it from notes